internal/cmd/kafka/command_topic_consume_onprem.go

```diff
@@ -26,11 +26,19 @@ func (c *authenticatedTopicCommand) newConsumeCommandOnPrem() *cobra.Command {
 		Long:  "Consume messages from a Kafka topic. Configuration and command guide: https://docs.confluent.io/confluent-cli/current/cp-produce-consume.html.\n\nTruncated message headers will be printed if they exist.",
 		Example: examples.BuildExampleString(
 			examples.Example{
-				Text: `Consume message from topic "my_topic" with SSL protocol and SSL verification enabled (providing certificate and private key).`,
-				Code: `confluent kafka topic consume my_topic --protocol SSL --bootstrap "localhost:19091" --ca-location my-cert.crt --cert-location client.pem --key-location client.key`},
+				Text: `Consume message from topic "my-topic" providing embedded Kafka REST Proxy endpoint.`,
```

Comment:
Looks like this change got reverted by accident?
```suggestion
				Text: `Consume messages from topic "my-topic" providing embedded Kafka REST Proxy endpoint.`,
```

Reply:
Good catch. Thanks @brianstrauch